tests/AwsLambda/UnitTests/AwsLambdaOpenTracerTests/TestUtil.cs

```diff
@@ -21,13 +21,21 @@ internal static LambdaSpan CreateSpan(string operationName, DateTimeOffset times
             span.SetContext(context);
             return span;
         }
-        internal static LambdaSpan CreateRootSpan(string operationName, DateTimeOffset timestamp, IDictionary<string, object> tags, string guid, ILogger logger = null)
+        internal static LambdaSpan CreateRootSpan(string operationName, DateTimeOffset timestamp, IDictionary<string, object> tags, string guid, ILogger logger = null, IFileManager fileManager = null)
         {
             if (logger == null)
             {
                 logger = new MockLogger();
             }
-            var rootSpan = new LambdaRootSpan(operationName, timestamp, tags, guid, new DataCollector(logger, false), new TransactionState(), new PrioritySamplingState(), new DistributedTracingState());
+
+            if (fileManager == null)
+            {
+                fileManager = new MockFileManager();
+            }
+
+
```

Comment:
Recommend only 1 empty line in between lines of code.

Reply:
Sorry about that, I've come to rely on my linter too much.